xliff/en.xliff

```diff
@@ -700,84 +696,84 @@ Make sure the barcode is well lit.</target>
       </trans-unit>
       <trans-unit id="product-detail.nutrition-table.for-serving">
-        <source>product-detail.nutrition-table.for-serving</source>
+        <source>Per serving</source>
         <note>No comment provided by engineer.</note>
-        <target xml:lang="en-GB" state="needs-translation">product-detail.nutrition-table.for-serving</target>
+        <target xml:lang="en-GB" state="needs-translation">Per serving</target>
       </trans-unit>
       <trans-unit id="product-detail.page-title.ingredients">
-        <source>product-detail.page-title.ingredients</source>
+        <source>Ingredients</source>
         <note>Product detail, ingredients</note>
-        <target xml:lang="en-GB" state="needs-translation">product-detail.page-title.ingredients</target>
+        <target xml:lang="en-GB" state="needs-translation">Ingredientss</target>
       </trans-unit>
       <trans-unit id="product-detail.page-title.nutrition">
-        <source>product-detail.page-title.nutrition</source>
+        <source>Nutrition</source>
         <note>Product detail, nutrition</note>
-        <target xml:lang="en-GB" state="needs-translation">product-detail.page-title.nutrition</target>
+        <target xml:lang="en-GB" state="needs-translation">Nutrition</target>
       </trans-unit>
       <trans-unit id="product-detail.page-title.nutrition-table">
-        <source>product-detail.page-title.nutrition-table</source>
+        <source>Nutrition table</source>
         <note>Product detail, nutrition table</note>
-        <target xml:lang="en-GB" state="needs-translation">product-detail.page-title.nutrition-table</target>
+        <target xml:lang="en-GB" state="needs-translation">Nutrition table</target>
       </trans-unit>
       <trans-unit id="product-detail.page-title.summary">
-        <source>product-detail.page-title.summary</source>
+        <source>Summary</source>
         <note>Product detail, summary</note>
-        <target xml:lang="en-GB" state="needs-translation">product-detail.page-title.summary</target>
+        <target xml:lang="en-GB" state="needs-translation">Summary</target>
       </trans-unit>
       <trans-unit id="user.alert.password-missing">
-        <source>user.alert.password-missing</source>
+        <source>Password is required</source>
         <note>No comment provided by engineer.</note>
-        <target xml:lang="en-GB" state="needs-translation">user.alert.password-missing</target>
+        <target xml:lang="en-GB" state="needs-translation">Password is required</target>
       </trans-unit>
       <trans-unit id="user.alert.username-missing">
-        <source>user.alert.username-missing</source>
+        <source>Username is required</source>
         <note>No comment provided by engineer.</note>
-        <target xml:lang="en-GB" state="needs-translation">user.alert.username-missing</target>
+        <target xml:lang="en-GB" state="needs-translation">Username is required</target>
       </trans-unit>
```

Comment:
These changes may lead to repetition of strings in Crowdin translations. I need to find a better solution, before we can merge this.